src/framework/mpe/mpetypes.h

```diff
@@ -476,6 +504,15 @@ struct ArticulationMeta
     duration_t overallDuration = 0;
     pitch_level_t overallPitchChangesRange = 0;
     dynamic_level_t overallDynamicChangesRange = 0;
+
+    bool operator==(const ArticulationMeta& other) const
+    {
+        return type == other.type
```

Comment:
What about timestamp? Should we add it to this comparison?

Reply:
sure, thanks